contracts/cw-proposal-single/src/contract.rs

```diff
@@ -263,15 +267,24 @@ pub fn execute_execute(
         None => vec![],
     };
 
-    let response = if !prop.msgs.is_empty() {
-        let execute_message = WasmMsg::Execute {
-            contract_addr: config.dao.to_string(),
-            msg: to_binary(&cw_core::msg::ExecuteMsg::ExecuteProposalHook { msgs: prop.msgs })?,
-            funds: vec![],
-        };
-        Response::<Empty>::default().add_message(execute_message)
-    } else {
-        Response::default()
+    let response = {
+        let res = Response::default();
```

Comment:
Same as earlier nit: my preference is to move this into the `match` statement below. What do you think about this?